lib/api/web-element/scoped-element.js

```diff
@@ -159,6 +159,16 @@ class ScopedWebElement {
 
       const parentElement = args[0];
 
+      // Check if parentElement is valid or not.
+      if (parentElement && parentElement.webElement) {
+        const parent = await parentElement.webElement;
+        if (parent === null) {
+          // The parent was of type WebElement but was not resolved.
+          // Supress further errors from find element commands.
+          suppressNotFoundErrors = true;
```

Comment:
we should probably directly return `null` here instead of setting `suppressNotFoundErrors` to `true` because continuing further does not make sense here.

Reply:
makes sense, updated.